package.json

```diff
@@ -54,18 +54,23 @@
     "jsdoc": "^3.6.3",
     "jsonlint2": "^1.7.1",
     "leche": "^2.3.0",
-    "mocha": "^5.2.0",
+    "mocha": "^6.0.0",
     "mockery": "^2.1.0",
     "nock": "^9.6.1",
-    "np": "^5.1.3",
+    "np": "^7.2.0",
+    "npm-force-resolutions": "0.0.3",
     "npm-upgrade": "^2.0.3",
-    "nyc": "^11.9.0",
+    "nyc": "^14.0.0",
     "shelljs": "^0.8.3",
     "shelljs-nodecli": "^0.1.1",
     "sinon": "^7.5.0"
   },
   "files": [
     "config",
     "lib"
-  ]
+  ],
+  "resolutions": {
+    "handlebars": "4.5.3",
+    "minimist": "1.2.3"
```

Comment:
What version of minimist has the security vulnerability? Looks like some of the packages in package-lock (e.g. json5) require minimist ^1.2.5. Could you set it to 1.2.5 instead?